Source/Particles/Collision/BackgroundMCCCollision.cpp

```diff
@@ -229,8 +252,8 @@ void BackgroundMCCCollision::doBackgroundCollisionsWithinTile
     amrex::Real vel_std = sqrt(PhysConst::kb * T_a / mass_a);
 
     // get collision parameters
-    auto scattering_processes = m_scattering_processes.data();
-    auto process_count = m_scattering_processes.size();
+    auto scattering_processes = m_scattering_processes_exe.data();
+    auto process_count        = m_scattering_processes_exe.size();
```

Comment:
Just to avoid signed-unsigned comparisons below in the loop
```suggestion
    int const process_count        = m_scattering_processes_exe.size();
```